packages/codebytes/src/index.tsx

```diff
@@ -75,6 +83,12 @@ export const CodeByteEditor: React.FC<CodeByteEditorProps> = ({
           onChange={(newText: string) => {
             setText(newText);
             onEdit?.(newText, language);
```

Comment:
why do we still need onedit and oncopy?

Reply:
we need these because there is forum specific logic that will be passed into these from next js; it differs from the default logic.

Reply:
(chatted about this with doug, and we kinda were on the same page that eventually will want to go back to a cleaner api for passing down callbacks, since we'll need them for next.js logic and tracking)

Reply:
whats the forum specific logic?